content/learn/book/getting-started/setup/_index.md

```diff
@@ -22,7 +22,7 @@ Once this is done, you should have the ```rustc``` compiler and the ```cargo```
 ### Install OS dependencies
 * [Linux](https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md)
 * Windows: Make sure to install [VS2019 build tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16)
-* MacOS: No dependencies here
+* MacOS: xcode-select --install or [Xcode](https://apps.apple.com/de/app/xcode/id497799835?mt=12)
```

Comment:
```suggestion
* MacOS: Install the Xcode command line tools with `xcode-select --install` or the [Xcode app](https://apps.apple.com/en/app/xcode/id497799835)
```

The command would look nicer in an inline code block and we should probably link to the English app store.

Reply:
Okay this makes this look better thank you.